src/main/java/TaskManager.java

```diff
@@ -0,0 +1,49 @@
+public class TaskManager {
+    private static final String LINEBREAK = System.lineSeparator();
+    private Task[] tasks = new Task[100];
+    private int taskCount;
+
+    public TaskManager() {
+        taskCount = 0;
+    }
+
+    public void displayTaskList() {
+        if (taskCount == 0) {
+            System.out.println("No task added yet!");
+        } else {
+            System.out.println("Here is your list at the moment:");
+            for (int i = 0; i < taskCount; i++) {
+                System.out.printf("%d. %s" + LINEBREAK, i + 1, tasks[i].toString());
+            }
+        }
+    }
+
+    public void addTask(String input, Action taskType) {
+        String[] parameters = Parser.parseTask(input, taskType);
+        switch (taskType) {
+        case TO_DO:
+            tasks[taskCount] = new ToDo(parameters[0]);
+            break;
+        case DEADLINE:
+            tasks[taskCount] = new Deadline(parameters[0], parameters[1]);
+            break;
+        case EVENT:
+            tasks[taskCount] = new Event(parameters[0], parameters[1]);
+            break;
+        }
+        System.out.println("Got it. I've added this task:");
```

Comment:
It is good to split into different functions, making it more concise compared to a large switch-case statements